lib/ansible/modules/cloud/misc/proxmox.py

```diff
@@ -333,6 +333,14 @@ def node_check(proxmox, node):
     return [True for nd in proxmox.nodes.get() if nd['node'] == node]
 
 
+def proxmox_version(proxmox):
+    apireturn = proxmox.version.get()
```

Comment:
You can also change the function as follows (and remove the `if`):
```suggestion
    apireturn = proxmox.version.get()
    return float(apireturn['version'])
```
This will also work for versions like `6.0-1`.